Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -197,6 +197,11 @@ int HardwareBreakpointManager::getAvailableLocation() {
   return (it - _locations.begin());
 }
 
+int HardwareBreakpointManager::findLocation(Address const &addr) {
```

Comment:
This does make the code a little less clean in the cases where this function was used, but probably worth it to remove a member function?